packages/lesswrong/lib/crud/withSingle.ts

```diff
@@ -1,4 +1,4 @@
-import { ApolloError, gql, useQuery, WatchQueryFetchPolicy } from '@apollo/client';
+import { ApolloClient, NormalizedCacheObject, ApolloError, gql, useQuery, WatchQueryFetchPolicy } from '@apollo/client';
```

Comment:
I'd appreciate some documentation/info in the PR description about `NormalizedCacheObject` and what's motivating the various changes with that

Reply:
It's not actually a change - our Apollo client already has the type `ApolloClient<NormalizedCacheObject>`, but everywhere we currently use it this can be automatically inferred. The only difference is that we're adding it as the type of an object property and it can't be inferred.